roles/confluent.test/molecule/plain-rhel/molecule.yml

```diff
@@ -100,6 +100,19 @@ provisioner:
       all:
         sasl_protocol: plain
 
+        # Testing secrets protection
+        secrets_protection_enabled: true
+
+        # Test secrets protection in kafka broker
+        kafka_broker_secrets_protection_config_path: "./etc/kafka/server.properties"
```

Comment:
use absolute paths when possible, this should be "/etc/kafka/server.properties"

Reply:
should the config path default to /etc/kafka/server.properties? Im guessing thats the most likely file to update right?

Reply:
I don't have it default to it because I have it set up so that if there is anything in the config_path variable, then secrets protection will run on that host. So it's empty by default.

Reply:
Ah I see how it works, you need to turn on secrets protection **and** specify a file for secrets protection to work w a given component. I'm going to make the claim that users might not know the config file paths and still want to enable this feature, so we should make it easier for them.

The pattern we use elsewhere is to have this in the defaults/main.yml
```
secrets_protection_enabled: true
kafka_broker_secrets_protection_enabled: "{{secrets_protection_enabled}}"
kafka_broker_secrets_protection_config_path : "{{kafka_broker.config_file}}"
kafka_broker_secrets_protection_config_params: []
```
And so the minimal set of variables a user would need to set would simply be:
```
secrets_protection_enabled: true
```
And then all passwords in server.props would get encrypted.

If they want to be more specific if would be 
```
secrets_protection_enabled: true
kafka_broker_secrets_protection_config_params: [advertised.listeners, broker.id]
```

And if they want secrets protection on across all the components except kafka it would be
```
secrets_protection_enabled: true
kafka_broker_secrets_protection_enabled: false
```